ntoskrnl/config/cmconfig.c

```diff
@@ -333,6 +333,11 @@ CmpInitializeHardwareConfiguration(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
     ULONG Disposition;
     UNICODE_STRING KeyName;
 
+    /* Set the alternative system architecture information */
+#if defined(SARCH_PC98)
+    SharedUserData->AlternativeArchitecture = NEC98x86;
```

Comment:
A experienced user suggest under the debate field to replace the thing with the macro
```suggestion
    SetNEC_98;
```

Reply:
Ok :)